plugin/geoip/README.md

```diff
@@ -0,0 +1,100 @@
+# geoip
+
+## Name
+*geoip* - Lookup maxmind geoip2 databases using the client IP, then add associated geoip data to the context request.
+
+## Description
+The *geoip* plugin add geo location data associated with the client IP, it allows you to configure a [geoIP2 maxmind database](https://dev.maxmind.com/geoip/docs/databases) to add the geo location data associated with the IP address.
+
+The data is added leveraging the [metadata](../metadata/README.md) plugin, values can then be retrieved using it as well, for example:
+
+```go
+import (
+    "strconv"
+    "github.com/coredns/coredns/plugin/metadata"
+)
+// ...
+if getLongitude := metadata.ValueFunc(ctx, "geoip/longitude"); getLongitude != nil {
+    if longitude, err := strconv.ParseFloat(getLongitude(), 64); err == nil {
+		// Do something useful with longitude.
+	}
+} else {
+    // The metadata label geoip/longitude for some reason, was not set.
+}
+// ...
+```
+
+## Databases
+Right now only databases providing the city schema such as `City` and `Enterprise` are supported, other databases types with different schemas are not supported yet. 
+
+You can download a free and public City database [here](https://www.maxmind.com/en/geoip2-city).
+
+## Syntax
+* **DBFILE** the mmdb database file path.
```

Comment:
move the DBFILE bullet below the syntax example

Reply:
Done